code/mapping/ext_modules/mapping_common/map.py

```
@@ -105,9 +106,71 @@ def get_entity_in_front_or_back(self, in_front=True) -> Optional[Entity]:
+        # converts lane box Rectangle to a shapely Polygon
+        lane_box_shapely = lane_box_shape.to_shapely(Transform2D.identity())
+
+        # creates intersection list of lane box with map entities
+        lane_box_intersection_entities = map_tree.query(
+            geo=lane_box_shapely, predicate="intersects"
+        )
+
+        # if list with lane box intersection is empty --> lane is free
+        if not lane_box_intersection_entities:
+            return True
+        return False
```

Ruff (0.8.2)

Return the condition `bool(not lane_box_intersection_entities)` directly

Replace with `return bool(not lane_box_intersection_entities)`

(SIM103)